### PR TITLE
[zh-cn]Sync to the latest version.

### PR DESCRIPTION
**Pull request by uos-ljtian**
Sync to the latest version.
Page path https://kubernetes.io/docs/tutorials/kubernetes-basics/scale/scale-intro/

### CODE REVIEW

File: content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: 运行应用程序的多个实例
 weight: 10
 description: |-
-  使用 kubectl 手动扩展现有应用程序。
+  使用 kubectl 手动伸缩现有应用程序。
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html:5]: scaling/scale 我们一般翻译为“扩缩”，扩容、缩容的简称。

File: content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html [hunk 2]
```diff
@@ -322,7 +322,7 @@ <h3>Scale Down</h3>
                -->
 
                <h3>缩小</h3>
-               <p>要将 Deployment 缩小到 2 个副本，请再次运行 <code>scale</code> 子命令：</p>
+               <p>要将 Deployment 缩小到2个副本，请再次运行 <code>scale</code> 子命令：</p>
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html:325]: 现在的空格留白是对的。

File: content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html [hunk 3]
```diff
@@ -266,7 +266,7 @@ <h3>扩伸部署</h3>
                <p>You can also view in the output of this command that there are 4 replicas now.</p>
                -->
 
-               <p>接下来，我们将 Deployment 扩伸到4个副本。我们将使用 <code>kubectl scale</code> 命令，后跟部署类型、名称和所需的实例数量：</p>
+               <p>接下来，我们将 Deployment 扩增到4个副本。我们将使用 <code>kubectl scale</code> 命令，后跟部署类型、名称和所需的实例数量：</p>
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html:269]: 现在的空格留白是对的。

File: content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html [hunk 4]
```diff
@@ -221,12 +221,12 @@ <h3>扩伸部署</h3>
                </ul>
                -->
 
-                <p>我们应该有 1 个 Pod。如果没有，请再次运行该命令。由此可见：</p>
+                <p>我们应该有1个 Pod。如果没有，请再次运行该命令。由此可见：</p>
```
Comment (tengqm) [inline content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html:224]: 现在的空格留白是对的。

File: content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html (hunk 1 above)
Comment (uos-ljtian) [reply content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html:5]: 好的，我再改一下

File: content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html (hunk 2 above)
Comment (uos-ljtian) [reply content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html:325]: 好的

File: content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html (hunk 4 above)
Comment (uos-ljtian) [reply content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html]: resolved

File: content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html (hunk 3 above)
Comment (uos-ljtian) [reply content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html]: resolved

**Comment by uos-ljtian** [comment]
> Please: 1) mark a feedback as "resolved" if you agreed to the suggestion and made the adjustments accordingly, this will make the next round review much easier; 2) always squash your PR into a single commit.

Thank you for the reminder, I have made the changes as requested.

File: content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html (hunk 3 above)
Comment (windsonsea) [reply content/zh-cn/docs/tutorials/kubernetes-basics/scale/scale-intro.html]: 每个 comment 有个按钮叫 `Resolve Conversation`。如果你按建议全部修改了，请点击那个按钮，逐条 resolved comment

**Comment by uos-ljtian** [comment]
This PR was closed while syncing the repository due to a conflict.
The conflicting PR is #43878

**Comment by uos-ljtian** [comment]
111

**Comment by uos-ljtian** [comment]
Since someone has already translated this page and it is basically consistent with my content, I don’t think it is necessary for me to continue to merge it.

**Pull request closed without merging**